deploy/docker-compose-weave/docker-compose.yml

```diff
@@ -13,6 +13,7 @@ services:
       - external
       - secure
       - internal
+      - backoffice
```

Comment:
So this seemed to be the simplest solution at the onset, where you would curl the health endpoint to get a json response detailing the health of the entire structure and a 200/500 response code for overall health. This seems to work fine for docker, but for k8s I guess it would be different after my conversation with Ilya [here](https://github.com/microservices-demo/front-end/pull/25)